src/dotnet/project.json

```diff
@@ -22,26 +22,34 @@
     },
     "NuGet.CommandLine.XPlat": "3.5.0-rc-1285",
     "Newtonsoft.Json": "7.0.1",
-    "System.Text.Encoding.CodePages": "4.0.1-rc2-24027",
-    "System.Diagnostics.FileVersionInfo": "4.0.0-rc2-24027",
+    "System.Text.Encoding.CodePages": "4.0.1-rc3-24113-00",
+    "System.Diagnostics.FileVersionInfo": "4.0.0-rc3-24113-00",
     "Microsoft.ApplicationInsights": "2.0.0",
-    "Microsoft.DotNet.ProjectModel": { "target": "project" },
-    "Microsoft.DotNet.Compiler.Common": { "target": "project" },
-    "Microsoft.DotNet.Cli.Utils": { "target": "project" },
-    "Microsoft.Extensions.Testing.Abstractions": { "target": "project" },
-    "Microsoft.NETCore.ConsoleHost": "1.0.0-rc2-24027",
-    "Microsoft.NETCore.TestHost": "1.0.0-rc2-24027",
+    "Microsoft.DotNet.ProjectModel": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.Compiler.Common": {
+      "target": "project"
+    },
+    "Microsoft.DotNet.Cli.Utils": {
+      "target": "project"
+    },
+    "Microsoft.Extensions.Testing.Abstractions": {
+      "target": "project"
+    },
+    "Microsoft.NETCore.ConsoleHost": "1.0.0-rc3-24113-00",
```

Comment:
Do we still need this dependency?  It brings in `CoreConsole.exe` on Windows, which I'm pretty sure we don't use.